packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.test.jsx

```diff
@@ -1002,12 +1002,13 @@ describe('DataSeriesFormItemModal', () => {
         cardConfig={timeSeriesCardConfig}
         editDataItem={{
           ...editTimeseriesDataItem,
-          aggregationMethod: ['min'],
+          aggregationMethod: 'min',
```

Comment:
aggregationMethod should not have been wrapped in an array, so unwrapping it caused it to actually appear correctly and the fallback `MIN` needed to be replaced with the correct `Min` in the expect below.